bitrise.yml

```diff
@@ -174,6 +174,7 @@ workflows:
           run_if: true
           inputs:
             - project_location: $IT_SOURCE_DIR
+            - gradle_options: "--stacktrace"
```

Comment:
This adds --stacktrace to all of our tests, to make our life easier when something fails.